client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -229,6 +232,27 @@ const CheckoutThankYou = React.createClass( {
 			);
 		}
 
+		if ( this.props.domainOnlySiteFlow && ! failedPurchases.length ) {
+			const [ purchase, domainName ] = findPurchaseAndDomain( purchases, isDomainRegistration );
+
+			return (
+				<Main className="checkout-thank-you">
+					{ this.renderConfirmationNotice() }
+
+					<ThankYouCard
+						name={ this.translate( 'Domain Registration' ) }
+						price={ purchase.displayPrice }
+						heading={ this.translate( 'Thank you for your purchase!' ) }
+						description={ this.translate( "That looks like a great domain. Now it's time to get it all set up." ) }
+						buttonUrl={ domainManagementEdit( domainName, domainName ) }
+						buttonText={ this.translate( 'Manage Your Domain' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **52** times:
`translate( 'Manage Your Domains' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-only-site-thank-you)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).